exporter/stats/stackdriver/stackdriver_test.go

```diff
@@ -24,16 +24,53 @@ import (
 	timestamp "github.com/golang/protobuf/ptypes/timestamp"
 	"go.opencensus.io/stats"
 	"go.opencensus.io/tag"
+	"google.golang.org/api/option"
 	distributionpb "google.golang.org/genproto/googleapis/api/distribution"
 	"google.golang.org/genproto/googleapis/api/label"
 	"google.golang.org/genproto/googleapis/api/metric"
 	metricpb "google.golang.org/genproto/googleapis/api/metric"
 	monitoredrespb "google.golang.org/genproto/googleapis/api/monitoredres"
 	monitoringpb "google.golang.org/genproto/googleapis/monitoring/v3"
+	"google.golang.org/grpc"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 )
 
+var authOptions = []option.ClientOption{option.WithGRPCConn(&grpc.ClientConn{})}
```

Comment:
@songy23 since you had mentioned/asked, here is how you can do it in Go, just create a fake gRPC client connection and that'll avoid looking for default credentials.

Reply:
I see, it's somewhat different from Stackdriver client in Java. Thanks!

Reply:
Here is what would mirror the Java code you posted
```go
const ts string
func (ts) Token() (*oauth2.Token, error) {
    return &Token{AccessToken: "fakeToken"}, nil
}
option.WithTokenTokenSource(ts{})
```
but notice how involved that is than `option.WithGRPCConn`, plus it requires an extra import of "golang.org/x/oauth2"